code/src/java/pcgen/core/PlayerCharacter.java

```diff
@@ -595,8 +595,26 @@ public PlayerCharacter(boolean load, Collection<Campaign> loadedCampaigns)
 		addSpellBook(new SpellBook(Globals.getDefaultSpellBook(), SpellBook.TYPE_KNOWN_SPELLS));
 		addSpellBook(new SpellBook(Constants.INNATE_SPELL_BOOK_NAME, SpellBook.TYPE_INNATE_SPELLS));
 	}
-	
-	private final void doFormulaSetup()
+
+	/**
+	 * Set the Weapon proficiency of one piece of Equipment to the same as the
+	 * Proficiency in another piece of Equipment.  For some bizarre reason, as
+	 * well as setting the proficiency,  this zeros out the Weight and cost of
+	 * the equipment.
+	 *
+	 * @param  equip  the Weapon to get the proficiency from
+	 * @param  eqm    the weapon to set the proficiency in
+	 */
+	static void setProf(final Equipment equip, final Equipment eqm)
```

Comment:
Please do not commit this.
This is separated out for a reason, and we should discuss the code design of putting it back into PlayerCharacter.